x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/all/helpers.test.ts

```diff
@@ -407,6 +407,14 @@ describe('#formatTimelineData', () => {
       });
     });
 
+    it('builds an object with no fields response', () => {
```

Comment:
@XavierM I added this test and the accompanying null check because the `parseEqlResponse` unit tests were feeding in events without a `fields` property. It certainly seems plausible for a hit not to have that property, hence the changes here, but I wanted to double-check it with you as well.